conda_smithy/templates/README.md.tmpl

```diff
@@ -14,24 +14,31 @@ Summary: {{ package.meta.about.summary }}
 Installing {{ package.name() }}
 ==========={{ '=' * package.name()|length }}
 
-Installing {{ package.name() }} from the conda-forge channel can be achieved by adding conda-forge to your channels with:
+Installing `{{ package.name() }}` from the `{{ channels.targets[0][0] }}` channel can be achieved by adding `{{ channels.targets[0][0] }}` to your channels with:
 
 ```
-conda config --add channels conda-forge
+conda config --add channels {{ channels.targets[0][0] }}
 ```
 
-Once the conda-forge channel has been enabled, {{ package.name() }} can be installed with:
+Once the `{{ channels.targets[0][0] }}` channel has been enabled, `{{ package.name() }}` can be installed with:
 
 ```
 conda install {{ package.name() }}
 ```
 
-It is possible to list all of the versions of {{ package.name() }} available on your platform with:
+Or, without updating your channel list, `{{ package.name() }}` can be installed with a single command:
```

Comment:
I'm not fond of the addition of more guidance on installation - we need to be careful not to replicate the conda docs in each and every feedstock. Would you mind removing the additional "Or, without updating your channel..." + the new install codeblock?